js_modules/dagster-ui/packages/ui-core/src/search/useGlobalSearch.tsx

```diff
@@ -53,6 +53,18 @@ export const linkToAssetTableWithOwnerFilter = (owner: string) => {
   })}`;
 };
 
+export const linkToAssetTableWithAssetOwnerFilter = (owner: AssetOwner) => {
```

Comment:
Not related to this PR, but it'd be nice to use a transformer on these JSON.stringify calls that  strips the __typename from the JSON strings

Reply:
in this case we actually need the __typename